File: src/Problems/Twosum/BoilerPlateFull/boilerplate.py

## Enter Code Here ##   
size_arr = int(input())
arr = []
for i in range(size_arr):
	arr.append(int(input()))
target = int(input())
num = twosum(arr,target)
for i in range(len(num)):
    print(num[i], end=" ")

File: src/Problems/Twosum/BoilerPlateFull/boilerplate.cpp

#include <bits/stdc++.h>

using namespace std;
                
## Enter Code Here ##

int main() {
    int size_arr;
cin>>size_arr;
vector<int> arr(size_arr);
for(int i=0;i<size_arr;i++){
 cin>>arr[i]; 
}
int target;
cin>>target;
    vector<int> num = twosum(arr,target);
    for(int i=0;i<num.size();i++){
 cout<<num[i]<<" "; 
 }
    return 0;
}
